docs/content/directory-structure.md

```diff
@@ -1,4 +1,4 @@
-> This [pre-alpha](https://docs.improbable.io/reference/13.1/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+> This [pre-alpha](https://docs.improbable.io/reference/13.1/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)
```

Comment:
With Improbadoc, we can use slightly fancier call-out boxes - I think the best one for all these cases would be:

<%(Callout type="warn" message="A warning")%>

Reply:
Changed all of those messages to fancier boxes.